packages/main/src/features/InputSuggestions.js

```diff
@@ -111,6 +111,79 @@ class Suggestions {
 		return false;
 	}
 
+	onPageUp(event) {
```

Comment:
Too much repetitive code. All the new methods have too much code in common. It needs to be extracted and reused.

Reply:
Done